fem/bilininteg.hpp

```diff
@@ -2602,6 +2608,31 @@ class VectorCurlCurlIntegrator: public BilinearFormIntegrator
                                    const Vector &elfun);
 };
 
+/** Class for integrating the bilinear form a(u,v) := (Q curl u, v) where Q is a
+    scalar coefficient, and v is a vector with components v_i in the L2 or H1 space.
```

Comment:
`where Q is an optional scalar coefficient`

Reply:
done!